test/qr.js

```diff
@@ -66,7 +67,8 @@ export async function generateQrCode(driver) {
  */
 export async function downloadQrCode(driver, title = 'automated test QR title') {
     const downloadQrButton = await driver.findElement(
-        ExtendedBy.submitButtonWithText('Download QR-Code'));
+        // TODO: Also test PNG download
+        ExtendedBy.submitButtonWithText('Download QR-Code-Plakat'));
```

Comment:
Lets create an issue for this then? Or did you already?

Reply:
Just did: #19